jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -176,9 +219,39 @@ static void afterClinitInitialization() {
 		/*[ELSE]*/
 		StringCoding.encode(new char[1], 0, 1);
 		/*[ENDIF]*/
+
 		/*[IF (Sidecar18-SE-OpenJ9|Sidecar19-SE)&!(PLATFORM-mz31|PLATFORM-mz64)]*/
```

Comment:
The code below would be easier to understand if this condition were repeated in the matching `[ENDIF]` comment.